sky/shell/platform/ios/FlutterView.mm

```diff
@@ -30,4 +39,25 @@ - (BOOL)enableInputClicksWhenVisible {
   return YES;
 }
 
+/* The container itself is not accessible but contains accessible elements. */
+- (BOOL)isAccessibilityElement {
+  return NO;
+}
+
+- (NSArray*)accessibleElements {
+  return _accessibleElements;
+}
+
+- (NSInteger)accessibilityElementCount {
```

Comment:
You may not need these accessors if you expose accessibleElements itself, they may be implied.